types/googlepay/index.d.ts

```diff
@@ -1146,6 +1172,45 @@ declare namespace google.payments.api {
         token: string;
     }
 
+    /**
+     * Definition of merchant provided offers that may be applicable to the
+     * current order.
+     */
+    interface OfferInfo {
+        /**
+         * List of merchant provided offers applicable to the current order.
+         */
+        offers: OfferDetail[];
+    }
+
+    /**
+     * Definition for each offer to be applied to this Payment Request.
+     */
+    interface OfferDetail {
+        /**
+         * Redemption code available for this transaction. This would be used to
```

Comment:
Maybe use more direct tenses to simplify the wording:
`This is used to identify the offer when the user decides...`

Reply:
Will apply in a separate PR